modules/header/react/lib/parts/Search.tsx

```diff
@@ -163,22 +163,24 @@ const iconStyle: React.CSSProperties = {
   left: spacing.xxs,
 };
 
-const SearchReset = styled('span')<Pick<SearchState, 'value'>>(
+const searchIconStyle: React.CSSProperties = {
```

Comment:
Do you mind using a styled `IconButton` for this so the treatment between this and `SearchReset` is the same?